app.js

```diff
@@ -88,18 +88,13 @@ async function processJob (referenceId) {
 }
 
 app.post('/storeData', (req, res) => {
-  const { associationIds, adminUnitId } = req.body
+  const { associationIds } = req.body
   if (!associationIds) {
     return res.status(400).send('Missing association ids in request body')
   }
-  if (!adminUnitId) {
```

Comment:
This isn't really needed anymore, so I removed the check here. This would allow us to remove it from the frontend as well.

Reply:
Removed that feature complely